Code/V2/src/themes/index.js

```diff
@@ -0,0 +1,27 @@
+/* 
+    script: color of the script on a key (number, letter, etc.)
+    key: color of a key in a neutral state
+    flashed: color of a key while flashed
+    selected: color of a key currently selected by the user
+*/
+
+export const defaultTheme = {
+    script: "#43a8c2",      //cyan      
+    key: "#ffffff",         //white       
+    flashed: "#ffff3c",     //yellow    
+    selected: "#ee8426",    //orange
+};
+
+export const darkTheme = {
```

Comment:
I completely agree with the message you sent me earlier! We will likely only be sticking to defaultTheme.

To keep our code clean (and only containing what we plan to use), let's remove "darkTheme" and "lightTheme" for now.